lib/philomena/images.ex

```diff
@@ -210,11 +210,15 @@ defmodule Philomena.Images do
 
   defp maybe_suggest_user_verification(_user), do: false
 
-  def count_pending_approvals() do
-    Image
-    |> where(hidden_from_users: false)
-    |> where(approved: false)
-    |> Repo.aggregate(:count)
+  def count_pending_approvals(user) do
+    if Canada.Can.can?(user, :approve, %Image{}) do
```

Comment:
For consistent, I would move this logic to the template (in web/templates/layout/_header_staff_links.html.slime) and leave this function alone.

Reply:
To be fair, that was my original plan, but the other queue shortcuts (dupes, etc) do this.